docs/docsite/rst/user_guide/playbooks_delegation.rst

```diff
@@ -77,7 +77,12 @@ To specify more arguments, use the following syntax::
             body: "{{ mail_body }}"
           run_once: True
 
-The `ansible_host` variable reflects the host a task is delegated to.
+.. warning::
+
+    While you can ``delegate_to`` a host that does not exist in inventory (by adding IP address, DNS name or whatever requirement the connection plugin has),
+    these won't be "true inventory hosts" and might cause issues. For exmample they don't inherit variables from the "all" group', which is where users commonly
+    set connection configuration parameters.  If really need to delegate_to a non inventory host mid play, you should use the M(add_host) action.
+
```

Comment:
Couple of translation issues here: 
- "while" is ambiguous to translation - it suggests time/during as well as the intended meaning here, "although".
- keeping the warning on a single line helps with translation updates
-  ```suggestion
    Although you can ``delegate_to`` a host that does not exist in inventory (by adding IP address, DNS name or whatever requirement the connection plugin has), doing so does not add the host to your inventory and might cause issues. Hosts delegated to in this way do not inherit variables from the "all" group', so variables like connection user and key are missing. If you must ``delegate_to`` a non-inventory host, use the :ref:`add host module <add_host_module>`.
```

Reply:
Tested the link locally - it works correctly. So if the suggestion is accurate, @bcoca if you accept it, I'll merge the PR as soon as CI runs.